### PR TITLE
Update Sphinx theme

### PR DESCRIPTION
**Pull request by arnaucasau**
### Summary
Updates to new Ecosystem theme.

Total file size increased (31.8 MB -> 151 MB)
index.html size increased (9 KB → 149 KB)
Build time increased (343s → 621s)

For more information: https://github.com/Qiskit/qiskit_sphinx_theme/issues/232

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -138,19 +138,13 @@
 # a list of builtin themes.
 #
 
-html_theme = "qiskit_sphinx_theme"
+html_theme = "qiskit-ecosystem"
+html_title = f"{project} {release}"
 
-html_logo = 'images/logo.png'
+#html_logo = 'images/logo.png'
```
Comment (Eric-Arellano) [inline docs/conf.py:144]: This should be deleted. It's better to not have commented out code: https://bytedev.medium.com/code-comment-anti-patterns-and-why-the-comment-you-just-wrote-is-probably-not-needed-919a92cf6758#:~:text=Avoid%20commenting%20out%20code%20and,to%20another%20team%20member%27s%20work.

File: docs/conf.py [hunk 2]
```diff
@@ -138,19 +138,13 @@
...
 #html_sidebars = {'**': ['globaltoc.html']}
```
Comment (Eric-Arellano) [inline docs/conf.py:145]: This can also be deleted

**Review by Eric-Arellano — APPROVED** [review]
Thanks!

File: docs/conf.py (hunk 1 above)
Comment (arnaucasau) [reply docs/conf.py:144]: Okay! Thank you Eric!

**Review by mtreinish — APPROVED** [review]
This looks fine to me is there a reason it's still in draft?

**Comment by arnaucasau** [comment]
Yes, the reason is the increasing on the time building the documentation, and the size of the files compared with the old theme. I will fix it soon in another PR, but in the meantime, I let it as a draft. I should have updated the comment in this PRs before with the information, but here you can see the details:

> Total file size increased (31.8 MB -> 151 MB)
> index.html size increased (9 KB → 149 KB)
> Build time increased (343s → 621s)

**Comment by Eric-Arellano** [comment]
Thanks @doichanj! Arnau doesn't have write access so will need a maintainer to merge.

**Pull request merged**